adhocteam/Head-Start-TTADP
# add all regions to activity report dropdown if user has central office

## Description of change
Added "All Regions" to the region select dropdown on the activity-reports page if the user has "central office" as their home region.

## How to test
Navigate to the landing page as a user with central office (14) as their home region. Notice that "All regions" is present in the dropdown. Switch between regions. The alerts and reports should change for that region.

## Issue(s)
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-199

## Checklists

### Every PR


- The region select is not keyboard navigable. (https://ocio-jira.acf.hhs.gov/browse/TTAHUB-197)

## N/A
- [n/a] API Documentation updated
- [n/a] Boundary diagram updated
- [n/a] Logical Data Model updated
- [n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy

- [n/a] Staging smoke test completed

### After merge/deploy
